integration/token/fungible/tests.go

```diff
@@ -221,6 +222,11 @@ func TestAll(network *integration.Infrastructure, auditor string) {
 	CheckAcceptedTransactions(network, "alice", "", AliceAcceptedTransactions[:1], nil, nil, nil)
 	CheckAcceptedTransactions(network, "alice", "", AliceAcceptedTransactions[:1], &t0, &t1, nil)
 
+	// Register an issuer wallet and issue with that wallet
```

Comment:
maybe split this comment into 3. register, gen crypto material, issue.

Reply:
addressed